### ISSUE TITLE
Tensorboard 2.12 profiling data can not be found if there is not directory named "plugins" 

### ISSUE THREAD

**Issue by Jiexin-Zheng** [tensorflow/tensorboard#6396]
### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version df7af2c6fc0e4c4a5b47aeae078bc7ad95777ffa

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=8, micro=16, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Linux', nodename='mlp-llga-dev-01', release='5.4.0-147-generic', version='#164-Ubuntu SMP Tue Mar 21 14:23:17 UTC 2023', machine='x86_64')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: True
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
WARNING: no installation among: ['tb-nightly', 'tensorboard', 'tensorflow-tensorboard']
WARNING: no installation among: ['tensorflow', 'tensorflow-gpu', 'tf-nightly', 'tf-nightly-2.0-preview', 'tf-nightly-gpu', 'tf-nightly-gpu-2.0-preview']
WARNING: no installation among: ['tensorflow-estimator', 'tensorflow-estimator-2.0-preview', 'tf-estimator-nightly']

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.12.1'

--- check: tensorflow_python_version
2023-05-18 10:39:35.692382: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: SSE4.1 SSE4.2 AVX AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
INFO: tensorflow.__version__: '2.12.0'
INFO: tensorflow.__git_version__: 'unknown'

--- check: tensorboard_data_server_version
INFO: data server binary: '/home/jiexinzh/anaconda3/envs/tf2/lib/python3.8/site-packages/tensorboard_data_server/bin/server'
INFO: data server binary version: b'rustboard 0.7.0'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/home/jiexinzh/anaconda3/envs/tf2/bin/tensorboard\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 32>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 32>
Loopback infos: [(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0)), (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'mlp-llga-dev-01'

--- check: stat_tensorboardinfo
INFO: directory: /tmp/.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=2883701, st_dev=64768, st_nlink=2, st_uid=1006, st_gid=1006, st_size=4096, st_atime=1683988214, st_mtime=1684406218, st_ctime=1684406218)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/home/jiexinzh/anaconda3/envs/tf2/lib/python3.8/site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py @ file:///croot/absl-py_1666362940888/work
aiohttp @ file:///croot/aiohttp_1670009560265/work
aiosignal @ file:///tmp/build/80754af9/aiosignal_1637843061372/work
appdirs==1.4.4
astunparse==1.6.3
async-timeout @ file:///opt/conda/conda-bld/async-timeout_1664876359750/work
attrs @ file:///croot/attrs_1668696182826/work
blinker==1.4
brotlipy==0.7.0
cachetools @ file:///tmp/build/80754af9/cachetools_1619597386817/work
certifi @ file:///croot/certifi_1683875369620/work/certifi
cffi @ file:///croot/cffi_1670423208954/work
charset-normalizer @ file:///tmp/build/80754af9/charset-normalizer_1630003229654/work
click @ file:///tmp/build/80754af9/click_1646038465422/work
cryptography @ file:///croot/cryptography_1677533068310/work
flatbuffers @ file:///home/ktietz/cip/python-flatbuffers_1634039120618/work
frozenlist @ file:///croot/frozenlist_1670004507010/work
gast @ file:///Users/ktietz/demo/mc3/conda-bld/gast_1628588903283/work
google-auth @ file:///opt/conda/conda-bld/google-auth_1646735974934/work
google-auth-oauthlib @ file:///opt/conda/conda-bld/google-auth-oauthlib_1660687784486/work
google-pasta @ file:///Users/ktietz/demo/mc3/conda-bld/google-pasta_1630577991354/work
grpcio @ file:///croot/grpc-suite_1681912592597/work
gviz-api==1.10.0
h5py @ file:///tmp/abs_4aewd3wzey/croots/recipe/h5py_1659091371897/work
idna @ file:///croot/idna_1666125576474/work
importlib-metadata @ file:///croot/importlib-metadata_1678997070253/work
keras @ file:///home/builder/mesters/opt/envs/tensorflow/conda-bld/keras_1682445665871/work/keras-2.12.0-py2.py3-none-any.whl
Keras-Preprocessing @ file:///tmp/build/80754af9/keras-preprocessing_1612283640596/work
Markdown @ file:///croot/markdown_1671541909495/work
MarkupSafe @ file:///opt/conda/conda-bld/markupsafe_1654597864307/work
mkl-fft==1.3.6
mkl-random @ file:///work/mkl/mkl_random_1682950433854/work
mkl-service==2.4.0
multidict @ file:///croot/multidict_1665674239670/work
numpy @ file:///work/mkl/numpy_and_numpy_base_1682952636358/work
oauthlib @ file:///croot/oauthlib_1679489621486/work
opt-einsum @ file:///tmp/build/80754af9/opt_einsum_1621500238896/work
packaging @ file:///croot/packaging_1678965309396/work
pip==23.0.1
pooch @ file:///tmp/build/80754af9/pooch_1623324770023/work
protobuf==3.20.3
pyasn1 @ file:///Users/ktietz/demo/mc3/conda-bld/pyasn1_1629708007385/work
pyasn1-modules==0.2.8
pycodestyle==2.10.0
pycparser @ file:///tmp/build/80754af9/pycparser_1636541352034/work
PyJWT @ file:///opt/conda/conda-bld/pyjwt_1657544592787/work
pyOpenSSL @ file:///croot/pyopenssl_1677607685877/work
PySocks @ file:///tmp/build/80754af9/pysocks_1605305779399/work
requests @ file:///croot/requests_1682607517574/work
requests-oauthlib==1.3.0
rsa @ file:///tmp/build/80754af9/rsa_1614366226499/work
scipy==1.10.1
setuptools==66.0.0
six @ file:///tmp/build/80754af9/six_1644875935023/work
tensorboard @ file:///home/builder/mesters/opt/envs/tensorflow/conda-bld/tensorboard_1682445826165/work/tensorboard-2.12.1-py3-none-any.whl
tensorboard-data-server @ file:///croot/tensorboard-data-server_1681498183723/work/tensorboard_data_server-0.7.0-py3-none-manylinux2014_x86_64.whl
tensorboard-plugin-profile==2.11.2
tensorboard-plugin-wit @ file:///home/builder/tkoch/workspace/tensorflow/tensorboard-plugin-wit_1658918494740/work/tensorboard_plugin_wit-1.8.1-py3-none-any.whl
tensorflow @ file:///home/builder/mesters/opt/envs/tensorflow/conda-bld/tensorflow-base_1682700874376/work/tensorflow_pkg/tensorflow-2.12.0-cp38-cp38-linux_x86_64.whl
tensorflow-estimator @ file:///home/builder/mesters/opt/envs/tensorflow/conda-bld/tensorflow-estimator_1682445976941/work/tensorflow_estimator-2.12.0-py2.py3-none-any.whl
termcolor @ file:///croot/termcolor_1668084651543/work
tomli==2.0.1
typing_extensions @ file:///croot/typing_extensions_1681939499988/work
urllib3 @ file:///croot/urllib3_1680254681959/work
Werkzeug @ file:///croot/werkzeug_1679489717957/work
wheel==0.38.4
wrapt @ file:///tmp/abs_c335821b-6e43-4504-9816-b1a52d3d3e1eel6uae8l/croots/recipe/wrapt_1657814400492/work
yarl @ file:///opt/conda/conda-bld/yarl_1661437085904/work
zipp @ file:///croot/zipp_1672387121353/work

``````

</details>

## Issue description
If my file structure under directory "plugins_dir" is:
![image](https://github.com/tensorflow/tensorboard/assets/52603083/195ac148-a131-4c42-b935-1ddba5b8c651)

and i run command "tensorboard --logdir plugins_dir/ --bind_all", it can work,
but if the file structure under directory "plugins_dir" is changed as:
![image](https://github.com/tensorflow/tensorboard/assets/52603083/50aa9ecd-9c66-496a-8cfe-1847e76f12f9)

where there is no directory named "plugins", it will fail even though i used the same command "tensorboard --logdir plugins_dir/ --bind_all":
![image](https://github.com/tensorflow/tensorboard/assets/52603083/0d4a48c1-120b-428c-8a70-93009fdb0f75)

Could you please explain this and fix it in the future?

**Comment by yatbear** [comment]
Hi @Jiexin-Zheng,

The profiler plugin actually lives in a different repo now. Can you please move this issue to https://github.com/tensorflow/profiler/issues? Thanks!

**Comment by Jiexin-Zheng** [comment]
> Hi @Jiexin-Zheng,
> 
> The profiler plugin actually lives in a different repo now. Can you please move this issue to https://github.com/tensorflow/profiler/issues? Thanks!

Sure, i have move it to [#issue627](https://github.com/tensorflow/profiler/issues/627)
